app/styles/main.scss

```diff
@@ -57,9 +56,12 @@ $footerHeight : 60px;
 footer {
   width: 100%;
   height: $footerHeight;
-  line-height: $footerHeight;
```

Comment:
Why did you remove line-height? This existed so the content in the footer was vertically centered...